tracing-appender/src/rolling.rs

```diff
@@ -67,20 +114,99 @@ pub fn hourly(
     RollingFileAppender::new(Rotation::HOURLY, directory, file_name_prefix)
 }
 
+/// Creates an daily, rolling file appender.
+///
+/// The appender returned by `rolling::daily` can be used with `non_blocking` to create
+/// a non-blocking, daily file appender.
+///
+/// The location of the log file will be specified the `directory` passed in.
+/// `file_name_prefix` specifies the prefix of the log file. It is appended by the current date.
```

Comment:
```suggestion
/// A `RollingFileAppender` will have a fixed rotation whose frequency is
/// defined by [`Rotation`](struct.Rotation.html). The `directory` and
/// `file_name_prefix` arguments determine the location and file name's _prefix_
/// of the log file. `RollingFileAppender` will automatically append the current date.
```
We should note how the current date and hour determined is determined. e.g., Is it local time, UTC, etc.?

Reply:
Yup will do.